### ISSUE TITLE
Remap (of any key) in settings tab not working

### ISSUE THREAD

**Issue by Semnodime** [brookhong/Surfingkeys#750]
## Error details
Modes in stack: Disabled,Normal
SurfingKeys: 0.9.31
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:61.0) Gecko/20100101 Firefox/61.0
URL: moz-extension://847884f7-06d9-4463-882f-01b7e9ad05b7/pages/options.html

## Context
I do not manage to successfully remap any key in the settings page.

Example to reproduce on my machine:
- Open surfingkeys settings tab
- Click on button «d» next to «Scroll a page down» with advanced settings disabled.
- Type «dd» or «ddd» or any other unused new mapping.
- Confirm by pressing either «Enter» or «Numpad-Enter»
Observation:
- Shortly blinking change being instantly reverted or not confirmed
Expectation:
- Permanent change of mapping configuration

**Comment by mindgitrwx** [comment]
Please give me your **configuration address**. configuration file must be as an raw javascript file.

For example, when you type 's' and 'e' in normal mode 
you would see 'Load settings from', and copy and paste your configuration address like

https://raw.githubusercontent.com/mindgitrwx/gitconventions/master/SurfingKeys-config-ko-dev.js

**Comment by sheoak** [comment]
I also have issues mapping keys. Note: I'm using bépo layout (french alternate keyboard).

**Follow me because it's strange:**

When I use the normal mode it works for some keys (c, t, s r…), but for some when I type the key, something else appears on the screen. For exemple "»" print a "9" when i enter it as a mapping. "^" gives me a "y" which is the qwerty key on the position, but "«" is not on the key "9"… Note that "^" is not a direct acces, it's a key modifier to type _â, û, î_… 

When I switch to advanced mode to try to fix this, nothing get saved. If I hit the save button it says loading settings from file:///undefined.

I tried to import my own local file. It didn't work (even so I have a message saying it's loaded when I press "save"). When I close chromium and come back, the file name is gone.

I'm on arch linux, using chromium 67.0.3396.87.
I installed surfinkeys from the store.
